bindings/python/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.10.4]
```

Comment:
Until the next release, we'll keep it like this:
```suggestion
## [Unreleased]
```

Can you also add a link to `#657` at the very end of this file? Otherwise, it doesn't render properly, thanks!